server/redux/reducers/user-reducer.js

```diff
@@ -37,14 +37,16 @@ const removeUser = userId => {
 
 /* --------------- THUNK ACTION CREATORS --------------- */
 
-const createAndEmitUser = socket => {
+const createAndEmitUser = (socket, user) => {
   return dispatch => {
     const userId = socket.id;
-    const user = Map(createUser(userId));
-    dispatch(addUser(user));
-    socket.on('sceneLoad', () => {
-      socket.emit('createUser', user);
-    });
+    const displayName = user.displayName;
+    const newUser = Map(createUser(userId, displayName));
+    dispatch(addUser(newUser));
+    console.log('User created');
+    if (socket.sceneLoaded) {
```

Comment:
What happens here if the scene hasn't loaded by this point? Is this a one shot deal where the user never loads if it's not loaded at this point?